packages/office-ui-fabric-react/src/components/Callout/CalloutContent.base.tsx

```diff
@@ -307,6 +294,22 @@ export class CalloutContentBase extends React.Component<ICalloutProps, ICalloutS
     this._setHeightOffsetEveryFrame();
   };
 
+  private _dismissOnClickOrScroll(ev: Event, preventDismiss?: boolean) {
```

Comment:
Would gladly entertain alternative names. It's private so not sure it matters much. _dismissOnEvent? _dismissOnMouseEvent?

Reply:
instead of passing in preventDismiss, I think it might make more sense to just not call this function. I'm not 100% sure why we didn't do this in the first place but it seems like the calling method should check this rather than having it passed through.

Reply:
As for name, I think that the name makes sense for now, we can always change it later.

Reply:
yeah, I should've caught that. I agree. 

---
In reply to: [288693923](https://github.com/OfficeDev/office-ui-fabric-react/pull/9260#discussion_r288693923) [](ancestors = 288693923)